ocaml/typing/typetexp.mli

```diff
@@ -47,7 +47,8 @@ end
 
 val valid_tyvar_name : string -> bool
 
-val transl_label : Parsetree.arg_label -> Types.arg_label
+val transl_label :
```

Comment:
Though many functions here do not have comments, it's good to add comments on any new additions.